mobile/lib/modules/backup/providers/backup.provider.dart

```diff
@@ -162,6 +162,10 @@ class BackupNotifier extends StateNotifier<BackUpState> {
         onlyAll: true,
         type: RequestType.common,
       );
+
+      if (list.isEmpty) {
```

Comment:
I don't think this will ever be empty, there will be that one default (isAll) album

Reply:
I know it doesn't, but Android throws an exception if this isn't in place. It's because the list is technically null until after the function finishes, even though there's an await.

Reply:
Hmm, how come I haven't encountered this issue yet

Reply:
Dunno, I'll send screenshots of the exception.